packages/legend-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx

```diff
@@ -379,6 +380,18 @@ const ServiceGeneralEditor = observer(() => {
               </div>
             )}
           </div>
+          {owners.length < MINIMUM_SERVICE_OWNERS &&
```

Comment:
I'm not a fan of this, but I don't think we could revisit this design decision again later.

Reply:
noted